app.js

```diff
@@ -9,6 +9,19 @@ app.get('/api/topics', getTopics);
 
 app.get('/api/articles/:id', getArticles)
 
+
+
+
+
+app.use((err, req, res, next) => {
+    const badCodes = ['22P02']
```

Comment:
This could be named slightly better, with it being production code. Maybe `errorCodes`